src/i18n/strings/tchap_translations.json

```diff
@@ -539,6 +539,11 @@
     "en": "Direct messages",
     "fr": "Messages directs"
   },
+  "Persons": {
+    "comments" : "Use persons to differenciate from Element label 'People' which we use in Tchap as 'Messages directs'",
```

Comment:
Maybe we can rollback the modification we have made to "People". This way we avoid to introduce a new label